frontend/src/features/admin-form/common/components/PreviewFormBanner/PreviewFormBanner.tsx

```diff
@@ -166,16 +166,32 @@ export const PreviewFormBanner = ({
         </Drawer>
       </Flex>
       {isPaymentEnabled && (
-        <Flex backgroundColor={'#3A3E46'}>
-          <Text
-            textStyle="body-2"
-            color="white"
-            ml="2rem"
-            mt="0.5rem"
-            mb="0.5rem"
-          >
-            Payments made in Form Preview mode will not reflect on Stripe.
-          </Text>
+        <Flex backgroundColor="neutral.900">
+          {process.env.SECRET_ENV === 'production' ? (
+            <Text
+              textStyle="body-2"
```

Comment:
Kinda minor but since the style props are being used twice shall we abstract them out?

```
const paymentTextProps: TextProps = {
    textStyle: 'body-2',
    color: 'white',
    ml: '2rem',
    mt: '0.5rem',
    mb: '0.5rem',
}

...

<Text {...paymentTextProps}> ...
...

<Text {...paymentTextProps}> ...

```

Reply:
thanks! done in [2d15015](https://github.com/opengovsg/FormSG/pull/6327/commits/2d1501511624ae94d597e3653f78df6290f999f4)

Reply:
omg accidentally committed the flipped the prod and non-prod conditional for this banner in the above commit, fixed in [8b1c6c3](https://github.com/opengovsg/FormSG/pull/6327/commits/8b1c6c31b0e4bc1afa75fe632796a8eae16ba9ba)